packages/Chem/src/utils/most-common-subs.ts

```diff
@@ -0,0 +1,73 @@
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+import {RDMol} from "@datagrok-libraries/chem-meta/src/rdkit-api";
+import {getRdKitModule} from "../package";
+import {getMolSafe} from './mol-creation_rdkit';
+
+
+export function getMCS(molecules: string, df: DG.DataFrame, compareElements: boolean, compareBonds: boolean): string {
+  let rdkit = getRdKitModule();
+
+  let molCol = df.columns.byName(molecules);
+  const nmolecules = molCol.length;
```

Comment:
We do not use Hungarian notation. It should be called "molCount" or "molecules" or "moleculeCount".